src/agent/src/netlink.rs

```diff
@@ -422,6 +477,141 @@ impl Handle {
         Ok(())
     }
 
+    async fn add_rules<I>(&mut self, list: I) -> Result<()>
+    where
+        I: IntoIterator<Item = Rule>,
+    {
+        use packet::nlas::rule::Nla;
```

Comment:
Please move the "use ..." to the header of the file.

Reply:
Thx. Nla exists in multiple module and is called multiple times in this file, such as packet::nlas::routes::Nla and packet::nlas::rule::Nla. To prevent conflict, import Nla here to make sense only for this function.